tests/test_gflow.py

```diff
@@ -156,7 +156,7 @@ def _graph5() -> GraphForTest:
     graph = nx.Graph()
     graph.add_nodes_from(nodes)
     graph.add_edges_from(edges)
-    meas_planes = {0: "XY", 1: "XY", 2: "ZX", 3: "YZ"}
+    meas_planes = {0: "XY", 1: "XY", 2: "XZ", 3: "YZ"}
```

Comment:
This error should have been caught by an AssertionError or another exception.

Reply:
I totally agree. Let's use pydantic.

Reply:
Note: data validation to `find_gflow` and others being added in #155 by @thierry-martinez , based on new data classes

Reply:
see below by @thierry-martinez 
https://github.com/TeamGraphix/graphix/pull/155/commits/778f709ecf45b8f9bc4f83d3fb0a1b977b0e886b